### PR TITLE
Leaderboard scene

### PR DESCRIPTION
**Pull request by IanKBovard**
Added leaderboard scene.
Adjust game speed.
Added condition for score submit.
Added Terms and conditions layer.
Saves formdata to local storage

### CODE REVIEW

File: src/pages/HackShackAttack/index.js [hunk 1]
```diff
@@ -37,43 +39,138 @@ class Game extends Phaser.Scene {
...
+    } else {
+      setError(true);
+    }
   };
+
+  return (
+    <Layer>
+      <Box alignSelf="end" pad="small">
+        <Link to={{ pathname: '/arcade' }}>
+          <Button icon={<FormPreviousLink />} label="Back to Arcade" />
```
Comment (britt6612) [inline src/pages/HackShackAttack/index.js:103]: I think just having the exit button ? Since we already have a back to arcade

Comment (IanKBovard) [reply src/pages/HackShackAttack/index.js]: in order for the game to start they must accept. i dont think closing there layer here is an option. going back to arcade seems, like the best choice.

Comment (britt6612) [reply src/pages/HackShackAttack/index.js]: got it makes sense!

**Comment by IanKBovard** [comment]
i also moved all the assets to the public folder. game should load faster after this merge

**Pull request merged**